### PR TITLE
Add note for upgrade issue when k8s service type is changed

### PR DESCRIPTION
**Pull request by harsh-px**
Signed-off-by: Harsh Desai <harsh@portworx.com>

### CODE REVIEW

**Comment by harsh-px** [comment]
+ @nathan-px  for formatting and structure

File: content/shared/upgrade/upgrade-nodeport-issue.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (zoxpx) [inline content/shared/upgrade/upgrade-nodeport-issue.md:1]: Should we put this under "common-errors" @ https://docs.portworx.com/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors/

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/upgrade/_index.md [hunk 2]
```diff
@@ -119,7 +121,7 @@ If you have nodes which have access to a private registry, follow [Step 2a: Push
 
 Otherwise, follow [Step 2b: Push directly to nodes using tarball](#step-2b-push-directly-to-nodes-using-tarball).
 
-#### Step 2a: Push to local registry server, accessible by air-gapped nodes
+#### Step 2a: Push to local registry server, accessible by airgapped nodes
```
Comment (zoxpx) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/upgrade/_index.md:124]: this change probably NOT required -- we consistently use `air-gapped` expression everywhere  (including http://install.portworx.com/air-gapped bootstrap)

Comment (harsh-px) [reply content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/upgrade/_index.md]: Done

File: content/shared/upgrade/upgrade-nodeport-issue.md (hunk 1 above)
Comment (harsh-px) [reply content/shared/upgrade/upgrade-nodeport-issue.md:1]: Yes, this is indeed in the common-errors page

This markdown is a hidden file for other files to embed (like a header file).

common-errors.md  in this PR embeds this as
```
{{% content "shared/upgrade/upgrade-nodeport-issue.md" %}}
```

**Pull request merged**